pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -128,13 +137,63 @@ func (p *prtbLifecycle) syncPRTB(binding *v3.ProjectRoleTemplateBinding) error {
 	return p.reconcileProjectAccessToGlobalResources(binding, roles)
 }
 
+func (p *prtbLifecycle) ensureProjectResourcesRoleBinding(binding *v3.ProjectRoleTemplateBinding) error {
+	clusterAndProject := strings.Split(binding.ProjectName, ":")
+	if len(clusterAndProject) != 2 {
+		return errors.Errorf("unexpected value for PRTB project name: %s (PRTB %s/%s)", binding.ProjectName, binding.Namespace, binding.Name)
+	}
+	projectID := clusterAndProject[1]
+	rb, err := p.m.rbLister.Get(projectID, projectresources.RoleName)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return errors.Wrapf(err, "failed to get rolebinding %s for namespace %s", projectresources.RoleName, projectID)
+	}
+	subject, err := pkgrbac.BuildSubjectFromRTB(binding)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get subject for prtb %s", binding.Name)
+	}
+	if rb == nil {
+		bindingID := pkgrbac.GetRTBLabel(binding.ObjectMeta)
+		rb, err = p.m.roleBindings.Create(&rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        projectresources.RoleName,
+				Namespace:   projectID,
+				Annotations: map[string]string{projectresources.AuthzAnnotation: bindingID},
```

Comment:
Realized this is a bug, different PRTBs are sharing this rolebinding so they can't all have the same label.